SimTracker/TrackAssociatorProducers/plugins/TrackAssociatorByPositionImpl.h

```diff
@@ -24,6 +24,10 @@
 
 #include <map>
 
+namespace edm {
+  class EDProductGetter;
+}
```

Comment:
forward definitions like this should be avoided unless there is a very good reason. Why not include the right header?

Reply:
removed and added 
#include "DataFormats/Common/interface/EDProductGetter.h"
insted